wenab-app/package.json

```diff
@@ -16,11 +16,20 @@
     "typescript": "^4.9.5",
     "web-vitals": "^2.1.4"
   },
+  "devDependencies": {
+    "cross-env": "7.0.3",
+    "dotenv-flow": "3.2.0",
+    "nswag": "13.18.2",
+    "prepend-file": "2.0.1",
+    "ts-node": "10.9.1"
+  },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "genClient": "ts-node ./tools/generateClient.ts",
+    "genClient:local": "cross-env NODE_ENV=localhost npm run genClient"
```

Comment:
You said this line is important - please elaborate

Reply:
`genClient` runs the code to generate the TS Wenab client. `genClient:local` does the same thing but first sets a variable to `localhost` so that it uses the Open API spec from the Wenab running on your machine instead of the prod one we will eventually have.